packages/react-native-windows-init/src/Cli.ts

```diff
@@ -13,10 +13,56 @@ import * as validUrl from 'valid-url';
 import * as prompts from 'prompts';
 import * as findUp from 'find-up';
 import * as chalk from 'chalk';
-
+import * as path from 'path';
 // @ts-ignore
 import * as Registry from 'npm-registry';
 
+import * as appInsights from 'applicationinsights';
+import {randomBytes} from 'crypto';
+
+/**
+ * Important:
+ * Do not use process.exit() in this script as it will prevent telemetry from being sent.
+ * See https://github.com/microsoft/ApplicationInsights-node.js/issues/580
+ */
+
+appInsights.setup('795006ca-cf54-40ee-8bc6-03deb91401c3');
+const telClient = appInsights.defaultClient;
+telClient.commonProperties.sessionId = randomBytes(16).toString('hex');
+if (process.env.RNW_CLI_TEST) {
+  telClient.commonProperties.isTest = process.env.RNW_CLI_TEST;
+}
+
+function sanitizeStackTrace(envelope: any /*context: any*/): boolean {
```

Comment:
duplciation between other.
If not able to share perhaps a `\\ CODE-SYNC: ..\..\foo.ts` comment to indicate fixes should be duplicated.

Reply:
thanks - I considered this but given that this lives in a separate package as the CLI, we'd have to create yet another package and make the cli and rnwinit dependent on it, seems like overkill for something as simple, lmk if you feel strongly about it

Reply:
in this PR this is the only instance of this functionality so I'll add a comment in the other change I have in the works